app/actions/local/group.ts

```diff
@@ -43,14 +52,18 @@ export const searchGroupsByNameInTeam = async (serverUrl: string, name: string,
     } catch (e) {
         // eslint-disable-next-line no-console
         console.log('searchGroupsByNameInTeam - ERROR', e);
-        return queryGroupsByNameInTeam(operator.database, name, teamId).fetch();
+        return queryGroupsByNameInTeam(database, name, teamId).fetch();
     }
 };
 
 export const searchGroupsByNameInChannel = async (serverUrl: string, name: string, channelId: string): Promise<GroupModel[]> => {
-    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
-    if (!operator) {
-        throw new Error(`${serverUrl} operator not found`);
+    let database;
+
+    try {
+        database = DatabaseManager.getServerDatabaseAndOperator(serverUrl).database;
+    } catch (e) {
+        forceLogoutIfNecessary(serverUrl, e as ClientErrorProps);
```

Comment:
I'm trying to understand what made you add this

Reply:
https://github.com/mattermost/mattermost-mobile/pull/6338#discussion_r901548836

What should the catch do in this case? If fetching the DB throws an error then it logs you out no?

Reply:
Nope, it will only log you out if a responde from a request returns dtatus code 401